DOI- 887 ew did reg to gh

royki opened the pull request:
### Summary
|             |   |
|-------------|---|
| Description | Migreate to GH Actions  |
| Jira issue  | DevOps - DOI-887  |

#### Type of request
<!--- Please delete options that are not relevant. -->
- [x] Migrated to GH Actions (non-breaking change which adds functionality)
- [x] Removed Travis CI (non-breaking change which adds functionality)

jrhender commented on .github/workflows/deploy.yml:
```diff
@@ -0,0 +1,67 @@
+      - name: Deploy on master branch
+        if: github.ref == 'refs/heads/master'
+        run: |
+          git checkout maste
```
Typo
```suggestion
          git checkout master
```

royki replied:
Fixed.

jrhender commented on .github/workflows/deploy.yml:
```diff
@@ -0,0 +1,67 @@
+
+      - name: Configure Git
+        run: |
+          git config --global user.email ${{ secrets.GH_EMAIL }}
+          git config --global user.name ${{ secrets.GH_USER }}
```
What did you end up using for `GH_EMAIL` and `GH_USER`? If secret we can message

royki replied:
GH_USER : `energyweb`
GH_EMAIL: `devops@energyweb.org`
There is possibility, it might fail. Let's see.

jrhender reviewed (changes requested):
I don't think that we want to actually change the `PULL_REQUEST_TEMPLATE`, right?

jrhender commented:
@royki Please remove changes to pull request template

royki commented:
> @royki Please remove changes to pull request template

Hi @jrhender which changes? You mean the remove template from status check ?

jrhender commented:
> > @royki Please remove changes to pull request template
> 
> Hi @jrhender which changes? You mean the remove template from status check ?

You committed changes to the pull request template. I think this should not be the case, right? 
![image](https://user-images.githubusercontent.com/4407464/125984791-e09b0a59-b7a2-4089-a193-d63a6dd34758.png)

jrhender commented:
@royki I think we still want the pull-request template (so it shouldn't be deleted) we just don't any changes to it in the PR.

royki commented:
> @royki I think we still want the pull-request template (so it shouldn't be deleted) we just don't any changes to it in the PR.

@jrhender 
aouuch. I added back the template :-D

jrhender reviewed (approved):
Looks good to me

jrhender commented:
@royki looks like the README changes introduces some codacity errors though: 
![image](https://user-images.githubusercontent.com/4407464/126538790-7d5756cd-e9d9-4ce1-aa03-210cb300fedf.png)

royki commented:
> @royki looks like the README changes introduces some codacity errors though:
> ![image](https://user-images.githubusercontent.com/4407464/126538790-7d5756cd-e9d9-4ce1-aa03-210cb300fedf.png)

These are auto generated.

knzeng-e commented on README.md:
```diff
@@ -258,78 +313,93 @@ const updateData: IUpdateData = {
 > The design goal of EW DID is to support different DID methods on Energy Web Chain. The resolver currently has the support for ERC1056 standard (`ethr` DID method). In future, we will have support for ERC725 standard which enables `erc725` DID method.
 
-* **Importing required modules**
+- **Importing required modules**
```
It seems that codacy complains about spacing here

```suggestion
-   **Importing required modules**
```

royki commented:
Hi @jrhender @knzeng-e ,
I reverted the README file. and it is passing the `Codacy Static Code Analysis`. 
Can you please check the `.travis.yml` conflict. I removed this from this branch. I am not able to go further. Please merge this after resolving the travis conflict.

knzeng-e reviewed (approved):
Hi @royki 
I resolved the travis conflict to finish the merge.
thank you for the work !

jrhender reviewed (approved):
Thanks for your work @knzeng-e and @royki. I will merge

knzeng-e commented:
:tada: This PR is included in version 0.6.0 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/ew-did-registry/v/0.6.0)
- [GitHub release](https://github.com/energywebfoundation/ew-did-registry/releases/tag/v0.6.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
